src/index.js

```diff
@@ -55,15 +55,9 @@ function prevalPlugin({types: t, template, transform, transformFromAst}) {
         path.replaceWith(replacement)
       },
       ImportDeclaration(path, {file: {opts: {filename}}}) {
-        const isPreval = looksLike(path, {
-          node: {
-            source: {
-              leadingComments(comments) {
-                return comments.some(isPrevalComment)
```

Comment:
Could we instead just make this: `return comments && comments.some(isPrevalComment)`?

Then we don't have to change anything else.